content/md/en/docs/reference/command-line-tools/srtool.md

```diff
@@ -1,9 +1,409 @@
+   ```bash
+   docker pull paritytech/srtool:1.62.0
+   ```
+
+   The command downloads and unpacks the image.
+
+## Get helper programs
+
+The Substrate runtime tool requires Docker.
+However, you don't need to know anything about using Docker to build a chain using `srtool`.
+Instead, you can use the following programs to interact with the Docker image.
+
+- [srtool-cli](https://github.com/chevdor/srtool-cli) provides a command-line interface to build the runtime using the `srtool` Docker image.
+
+- [srtool-app](https://gitlab.com/chevdor/srtool-app) provides a simple graphical user interface for building the runtime using the `srtool` Docker image.
+
+- [srtool-actions](https://github.com/chevdor/srtool-actions) provides Github actions to integrate builds produced using the `srtool` image with your CI/CD pipeline.
+
+- [subwasm](https://github.com/chevdor/subwasm) provides command-line options for working with the metadata and WebAssembly runtime built using `srtool`.
+
+## srtool-cli
+
+You can use the `srtool` command-line interface to build the WebAssembly runtime using the srtool Docker image.
+The `srtool` command-line interface enables the same source code to consistently produce the same WebAssembly binary.
+
+### Basic command usage
+
+The basic syntax for running `srtool` commands is:
+
+```text
+srtool [options] [subcommand]
+```
+
+### Options
+
+You can use the following command-line options with the `srtool` command.
+
+| Option | Description |
+| -------| ----------- |
+| `-h`, `--help`  | Displays usage information. |
+| `i`, `--image <image>` | Specifies an alternative image. Be sure to specify an image that is compatible with the default `paritytech/srtool` image. You should note that specifying an image other than the `paritytech/srtool` image might not produce the same deterministic result that the `paritytech/srtool` image produces. |
+| `-n`, `--no-cache` | Specifies that you don't want to use the tag value from the local cache. |
+| `-V`, `--version` | Displays version information.   |
+
+### Subcommands
+
+You can use the following subcommands with the `srtool` command.
+
+| Command | Description |
+| ------- | ----------- |
+| `build` | Starts a new `srtool` container to build your runtime. |
+| `help` | Displays usage information for `srtool` or for a specified subcommand. |
+| `info` | Displays information about the `srtool` container and your repository. | 
+| `pull` | Pulls the `srtool` image without running anything else. |
+| `version` | Displays version information for the `srtool` container. Use `--version` if you want version information for the `srtool-cli` executable.
+
+### Examples
+
+To get version information for the `srtool` Docker image, run the following command:
+
+```bash
+srtool version
+```
+
+The command displays output similar to the following:
+
+```text
+{
+  "name": "srtool",
+  "version": "0.9.21",
+  "rustc": "1.62.0",
+  "subwasm": "0.18.0",
+  "tera": "0.2.1",
+  "toml": "0.2.1"
+}
+```
```

Comment:
Beware, if you show a run of each sub command, this may quick become deprecated.
I do that in the main repo because I use Asciidoc and I have scripts to automatically produce the output and generate the doc. Here I would simply put a link towards the repo, that will simplify the maintenance for you.

Reply:
Warning taken. I think I'll leave the examples in for now (since I was able to walk through all of them and it is generally nice to provide sampleinput and resulting output), but I can remove them/replace them with links if maintenance becomes as issue.